<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EffectsEditorWidget</class>
 <widget class="QWidget" name="EffectsEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="modifierAddB">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="typeCombo">
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Magic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ice</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nautre</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="modifiersLabel">
     <property name="text">
      <string>Modifiers</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QPushButton" name="addB">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="modifierRemoveB">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="durationLabel">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QSpinBox" name="durationEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="idEdit"/>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QListWidget" name="modifiersList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
